libraries/engage/page/widgets/HeroBanner/hooks.js

```diff
@@ -4,14 +4,14 @@ import { resolveBorderRadiusFromWidgetConfig } from '../../helpers';
 /**
  * @typedef {Object} HeroBanner
  * @property {string} text Banner text content
- * @property {Object} backgroundImage Banner background image
```

Comment:
this type is used to create the return type of the `useWidget` hook. shape of the widget config didn't change, to please revert this.